theano/gpuarray/opt.py

```diff
@@ -1094,7 +1097,9 @@ def local_inplace_sparseblockouter(node):
 @register_opt2([AbstractConv2d,
                 AbstractConv2d_gradWeights,
                 AbstractConv2d_gradInputs], 'fast_compile')
-def local_lift_abstractconv2d(node, context_name, inputs):
+def local_lift_abstractconv2d(op, context_name, inputs):
+
+    node = op.make_node(*inputs)
```

Comment:
Don't do that. It is not useful and and would be slow. Modify line 1111 instead.